packages/gatsby-theme-project-portal/src/layouts/ProjectDetailPage.tsx

```diff
@@ -28,7 +28,7 @@ export const ProjectDetailPage: FunctionComponent<
   } = props
 
   return (
-    <Layout title={project.title} description={project.summary}>
```

Comment:
Quick question, so what happens to these descriptions and title?

Reply:
Looking at it overall, doesnt seem like we need title and description anymore right? Should we clean it up

Reply:
The Layout props title and description are passed to the Head component and merged into the metadata, which is no longer handled by the Layout. I've taken out the extra props on the Layout components – thanks for catching this!